android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetOngoingConferenceService.java

```diff
@@ -99,22 +94,18 @@ public static void doLaunch(Context context, HashMap<String, Object> extraData)
         }
     }
 
-    public static Object onRequestPermissionsResult(int requestCode, Context context) {
-        if (permissionsPromises.containsKey(requestCode)) {
-
-            // If request is cancelled, the result arrays are empty.
-            Promise permissionsPromise = permissionsPromises.get(requestCode);
+    private static PermissionListener getNotificationPermissionListener(Context context, HashMap<String, Object> extraData) {
+        return new PermissionListener() {
+            @Override
+            public boolean onRequestPermissionsResult(int i, String[] strings, int[] ints) {
+                if (ContextCompat.checkSelfPermission(context, POST_NOTIFICATIONS) == PackageManager.PERMISSION_GRANTED) {
```

Comment:
This might work, but it's not the usual way to go about it. Check the result in the `ints` arrary (also use a better name please).